Preprocessing/adcpBinMappingPP.m

```diff
@@ -81,16 +81,32 @@
     % while beams 2 and 3 get further away. When roll is positive beam 3 is
     % closer to the surface while beam 2 gets further away.
     %
+
+    isUpwardLooking = true;
```

Comment:
As suggested by Hugo, instead of declaring the variable above and then later declare signs for both pitch and roll while only pitch changes sign depending on whether the ADCP is facing up or down, you could just:
- declare pitchSign = 1 here, 
- then pitchSign = -1 in the if statement below,
- finally just use pitchSign in the equations line 123 and 124 below